src/pages/pass-finalisation/cat-c/components/code-78/__tests__/code-78.spec.ts

```diff
@@ -66,49 +64,5 @@ describe('code78Component', () => {
       });
     });
 
-    describe('showAutomaticBanner', () => {
-      it('should show the automatic banner when it is valid', () => {
-        component.ngOnChanges();
-        component.category = TestCategory.C;
-        component.transmission = TransmissionType.Automatic;
-        component.form.get(Code78Component.fieldName).setValue('yes');
-        fixture.detectChanges();
-        expect(component.shouldShowAutomaticBanner()).toEqual(true);
-      });
-    });
-
-    describe('should show manual banner', () => {
-      it('should show the manual banner when it is valid', () => {
-        component.ngOnChanges();
-        component.category = TestCategory.C;
-        component.transmission = TransmissionType.Manual;
-        component.form.get(Code78Component.fieldName).setValue('yes');
-        fixture.detectChanges();
-        expect(component.shouldShowManualBanner()).toEqual(true);
-      });
-    });
-
-    describe('shouldHideBanner', () => {
-      it('should hide banner when only transmission is selected', () => {
-        component.ngOnChanges();
-        component.transmission = TransmissionType.Manual;
-        fixture.detectChanges();
-        expect(component.shouldHideBanner()).toEqual(true);
-      });
-
-      it('should hide banner when only code78 is selected', () => {
```

Comment:
could you copy this test and the one below to the pass completion page - You deleted 5 tests here but only moved 3 to the other spec file